src/components/modal-flow/ModalFlow.js

```diff
@@ -0,0 +1,373 @@
+import React, { Component } from 'react';
```

Comment:
Opinion: we could have made a component per "variant", so that the code is easier to reason about and is more scalable as other variants are added in the future. We don't need to do that now, but the note remains.

Reply:
Ok, I see your point. I'll do that if we add other variants in the future.